scripts/airflow3/README.md

```diff
@@ -1,50 +1,52 @@
 # Run Airflow3 Locally
 
-This guide walks you through setting up Apache Airflow 3 locally using Hatch and a Postgres container. You'll run a Postgres container to use it as a backend database for Airflow and set up the necessary environment to run Airflow.
+This guide will walk you through the process of setting up Apache Airflow 3 locally using pip. You can choose either SQLite or Postgres as the database backend for Airflow. In this setup, we'll be using a Postgres profile to run Cosmos DAGs.
 
-## 1. Setup Postgres Container (Optional)
+## 1. Setup Postgres Container
 
-We'll use PostgreSQL as the backend. The following command will pull the official Postgres image, create a container named postgres, and expose the necessary ports.
+By default, SQLite will be used unless you update the AIRFLOW__DATABASE__SQL_ALCHEMY_CONN environment variable to point to PostgreSQL. The following command will pull the official Postgres image, create a container named postgres, and expose the required ports.
+
+### 1.1 Pull Postgres Image
 
 ```commandline
 docker run --name postgres -p 5432:5432 -p 5433:5433 -e POSTGRES_PASSWORD=postgres postgres
 ```
 
-## 2. Access the PostgreSQL Console and Create the Database
+### 1.2 Access the PostgreSQL Console and Create the Database
 
 Now that the PostgreSQL container is running, you can connect to it via the command line using psql
 
 ```commandline
 psql --u postgres
 ```
 
-### Create the Database for Airflow
+### 1.3 Create the Database for Airflow
 
 Once you're inside the psql interactive terminal, you can create a new database that Airflow will use.
 
 ```commandline
 CREATE DATABASE airflow_db;
 ```
 
-## 3. Setup Virtual Environment for Airflow3
+## 2. Setup Virtual Environment for Airflow3
 
 With your Postgres container running and your database set up, you need to configure the virtual environment for Airflow3.
```

Comment:
```suggestion
You need to configure the virtual environment for Airflow3.
```

I guess Postgres is not related to Airflow 3 setup, hence, we could skip that line.